vowpalwabbit/gd.cc

```diff
@@ -17,6 +17,7 @@
 #endif
 #endif
 
+#include <string>
```

Comment:
might not be needed since only the include and the using got added on this file.

Reply:
Yes.  Removed.